qe_metrics/cli.py

```diff
@@ -0,0 +1,70 @@
+import os
+from typing import Dict, Any
+
+import click
+
+from simple_logger.logger import get_logger
+
+from qe_metrics.libs.database import Database
+from qe_metrics.libs.jira import Jira
+
+
+@click.command()
+@click.option(
+    "--services-file",
+    default=os.environ.get("QE_METRICS_SERVICES", "services.yaml"),
+    help="Defines the path to the services file.",
+    type=click.Path(exists=True),
+)
+@click.option(
+    "--creds-file",
+    default=os.environ.get("QE_METRICS_CREDS", "creds.yaml"),
+    help="Defines the path to the file holding database and Jira credentials.",
+    type=click.Path(exists=True),
+)
+@click.option(
+    "--pdb",
+    help="Drop to `ipdb` shell on exception",
+    is_flag=True,
+)
+@click.option(
+    "--verbose",
+    is_flag=True,
+    help="Verbose output of database connection.",
+    type=click.BOOL,
+)
+def main(**kwargs: Dict[str, Any]) -> None:
+    """Gather QE Metrics"""
+
+    # Remove the pdb option from the kwargs
+    kwargs.pop("pdb", None)
+
+    # Adding noqa: F841 to ignore the unused variable until next PR, otherwise pre-commit will fail
+    database = Database(creds_file=str(kwargs["creds_file"]))  # noqa: F841
```

Comment:
no need to cast as `str`

Reply:
It will throw a mypy error otherwise because the `creds_file` argument should be a string and technically the kwargs value could be anything. We can cast it as a `str` or add a `#type: ignore` comment. I am fine with either, do you have a preference?

Reply:
better option is to not get kwargs, get what you need.

Reply:
@myakove so do you think we should stop using kwargs considering we don't have many arguments from click being passed anyway?

Reply:
Yes